reference/viewpoint.md

```diff
@@ -32,10 +33,10 @@ plane. This plane is parallel to the projection plane for the 3D display in the
 main window. The near field determines the precision of the OpenGL depth buffer.
 A too small value may cause depth fighting between overlaid polygons, resulting
 in random polygon overlaps. The far clipping plane is parallel to the near
-clipping plane and is defined at an infinite distance from the camera. The far
-clipping plane distance cannot be modified.
+clipping plane and is defined by the `far` field. The far clipping plane can be
+set at an infinite distance by setting the `far` field to 0.
```

Comment:
It seems to be important to explain why the far plane can be important to set, i.e., to filter-out far objects in order to increase the rendering performance.